<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XiVO Client Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../xivoclient.qrc">
    <normaloff>:/images/xivoicon.png</normaloff>:/images/xivoicon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="user">
      <attribute name="title">
       <string>User</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_8">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="language_label">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="language"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="style"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="style_label">
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_9">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="auto_reconnect">
           <property name="text">
            <string>Auto-reconnect</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="startup_connect">
           <property name="text">
            <string>Connect at startup</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="start_systrayed">
           <property name="text">
            <string>Start systrayed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_10">
         <item row="0" column="0">
          <widget class="QCheckBox" name="show_window_on_call">
           <property name="text">
            <string>Show window when calling from external application</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="enable_clipboard">
           <property name="text">
            <string>Enable clipboard (GNU/Linux only)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="reset_docks">
           <property name="text">
            <string>Reset dock positions</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="features">
      <attribute name="title">
       <string>Features</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolBox" name="features_chooser">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(255, 85, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="presence">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>456</width>
            <height>230</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Presence</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="0">
            <widget class="QCheckBox" name="enable_presence_reporting">
             <property name="text">
              <string>Presence reporting</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="screen_popup">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>282</width>
            <height>94</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Screen popup</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_5">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="3" column="0">
            <widget class="QLabel" name="max_screen_popups_label">
             <property name="text">
              <string>Max screen popups</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="max_screen_popups">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>99</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="enable_screen_popup">
             <property name="text">
              <string>Screen Popup</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="enable_auto_open_url">
             <property name="text">
              <string>Allow automatic URL opening</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="history">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>148</width>
            <height>38</height>
           </rect>
          </property>
          <attribute name="label">
           <string>History</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="history_size_label">
             <property name="text">
              <string>History size</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="history_size">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="contacts">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>273</width>
            <height>44</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Contacts</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="max_contacts_displayed_label">
             <property name="text">
              <string>Maximum of contacts displayed</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="max_contacts_displayed">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>30</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="queue_list">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>456</width>
            <height>230</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Queue list</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QFormLayout" name="formLayout_7">
             <item row="0" column="0">
              <widget class="QCheckBox" name="show_longest_wait_time">
               <property name="text">
                <string>Show longest wait time</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout" columnstretch="9,0,0,0">
             <item row="0" column="1">
              <widget class="QLabel" name="middle_threshold_label">
               <property name="text">
                <string>Orange</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="high_threshold_label">
               <property name="text">
                <string>Red</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="longest_wait_time_label">
               <property name="text">
                <string>Longest wait time thresholds</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="longest_wait_time_green">
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="longest_wait_time_orange">
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="waiting_calls_label">
               <property name="text">
                <string>Waiting calls thresholds</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="waiting_calls_green">
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QSpinBox" name="waiting_calls_orange">
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLabel" name="waiting_calls_unit_label">
               <property name="text">
                <string>calls</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLabel" name="longest_wait_time_unit_label">
               <property name="text">
                <string>seconds</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="queue_members">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>188</width>
            <height>37</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Queue Members</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_12">
           <item row="0" column="0">
            <widget class="QCheckBox" name="hide_unlogged_agents">
             <property name="text">
              <string>Hide unlogged agents</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="administration">
      <attribute name="title">
       <string>Administration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,9,0,3,1">
         <item row="0" column="0">
          <widget class="QLabel" name="server_label">
           <property name="text">
            <string>Server</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="server">
           <property name="text">
            <string>demo.xivo.fr</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="port_label">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="encrypted">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../xivoclient.qrc">
             <normaloff>:/images/padlock.png</normaloff>:/images/padlock.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="backup_server_label">
           <property name="text">
            <string>Backup server</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="backup_server"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="backup_port_label">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="backup_encrypted">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../xivoclient.qrc">
             <normaloff>:/images/padlock.png</normaloff>:/images/padlock.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="backup_port">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>5003</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="port">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>5003</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="show_agent_options">
           <property name="text">
            <string>Show agent options</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="auto_reconnect_interval_label">
         <property name="text">
          <string>Auto-reconnect interval</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="auto_reconnect_interval">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3600</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="keepalive_interval_label">
         <property name="text">
          <string>Keepalive interval</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="keepalive_interval">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3600</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="allow_multiple_instances">
         <property name="text">
          <string>Allow multiple instances</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="show_displayprofile">
         <property name="text">
          <string>Display the configuration profile</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../xivoclient.qrc"/>
  <include location="../../xivoclient.qrc"/>
 </resources>
 <connections/>
</ui>
